Components/Pages/Goals.js

```diff
@@ -87,24 +94,22 @@ const Goals = ({ navigation }) => {
     setNotification({ notification: notification });
   };
 
-  const sendPushNotification = async () => {
-    const message = {
-      to: expoPushToken,
-      sound: "default",
-      title: "Push",
-      body: "🏴‍☠️ Arg! Walk the plank",
-      data: { data: "goes here" },
-      _displayInForeground: true,
-    };
-    const response = await fetch("https://exp.host/--/api/v2/push/send", {
-      method: "POST",
-      headers: {
-        "Accept": "application/json",
-        "Accept-encoding": "gzip, deflate",
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify(message),
-    });
+
+  const attachToken = async () => {
+    if (!notificationsEnabled) {
+      if (expoPushToken) {
+        await fetch(NOTIFICATION_URI, {
+          method: "POST",
+          headers: {
+            "Accept": "application/json",
+            "Accept-encoding": "gzip, deflate",
+            "Content-Type": "application/json",
+          },
+          body: { expoPushToken },
+        });
```

Comment:
this is working but i noticed the post request doesnt have a response. In a future iteration, the server should respond with something after this post request as a check